lxd/instance/drivers/driver_qemu.go

```diff
@@ -3376,19 +3376,21 @@ func (d *qemu) addRootDriveConfig(mountInfo *storagePools.MountInfo, bootIndexes
 	if d.storagePool.Driver().Info().Remote {
 		vol := d.storagePool.GetVolume(storageDrivers.VolumeTypeVM, storageDrivers.ContentTypeBlock, project.Instance(d.project.Name, d.name), nil)
 
-		config := d.storagePool.ToAPI().Config
+		if shared.ValueInSlice(d.storagePool.Driver().Info().Name, []string{"ceph", "cephfs"}) {
```

Comment:
Lets land this in a separate PR as well. Thanks

Reply:
Moved with https://github.com/canonical/lxd/pull/12569